packages/code-editor/src/customTabs/Markdown/index.js

```diff
@@ -131,49 +106,128 @@ export default class Markdown extends Component {
     `${t('project.features.nowFeatures')}<b>${isPublic ? t('project.features.public') : t('project.features.private')}</b> ${isPublic ? t('project.features.publicDescription') : t('project.features.privateDescription')}`)
   }
 
+  openLink = link => {
+    fileOps.current.openLink(link)
+  }
+
+  scrollTo = id => {
+    const el = window.document.querySelector(id)
+    if (!el) {
+      return
+    }
+    el.scrollIntoViewIfNeeded()
+    el.style.background = 'var(--color-secondary)'
+    setTimeout(() => {
+      el.style.background = ''
```

Comment:
can briefly explain why the element needs a background color and then remove it again